tekton/tasks/run-sanity-test-task.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: run-basic-sanity-check
+spec:
+  stepTemplate:
+    volumeMounts:
+      - name: project
+        mountPath: /workspace/source/image
+      - name: directory
+        mountPath: /workspace/source/directory
+  results:
+    - name: testResult
+      description: the result of sanity test
+  steps:
+    - name: clone-git-repo
```

Comment:
This is the previous approach, we now have docker image that has git together with conftest in it. Look here:
https://github.com/redhat-appstudio/build-definitions/blob/main/tasks/sanity-label-check.yaml
how it is used. I would recommend changing those and use **quay.io/redhat-appstudio/hacbs-test:stable** instead.

Reply:
@jsztuka Thank you! Let me update them.

Reply:
@jsztuka I tried image `quay.io/redhat-appstudio/hacbs-test:stable`, but I can't find `git` in it
besides, I find the two policies for label was built into the image, is it designed like this? Should we also build the policy for sign check into the image? Do you have the Dockerfile this image is built from? Thank you!
```
hongliu@hongliu1-mac HAS % docker run --rm -it quay.io/redhat-appstudio/hacbs-test:stable 
[root@67855c8513f7 /]# which git
/usr/bin/which: no git in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
[root@67855c8513f7 /]# ls /project/
image
[root@67855c8513f7 project]# ls /project/image/policy/
deprecated-labels.rego	required-labels.rego
```

Reply:
It has not git and its on purpose, the policies should be contained within the image. You can add those at here:
https://github.com/redhat-appstudio/hacbs-test/tree/main/policies/image/policy
Once you have the policies in image you dont need to use git at all.

Reply:
Thanks, PR is committed there https://github.com/redhat-appstudio/hacbs-test/pull/1